styleguide/components/00-settings/_colors.scss

```diff
@@ -20,14 +20,16 @@ $tints: (
   1: 10%,
   2: 25%,
   3: 50%,
-  -1: 75%,
-  -2: 50%,
-  -3: 25%,
-  -4: 10%,
-  -5: 6%,
-  -6: 2%,
+  -1: .75,
+  -2: .5,
+  -3: .25,
+  -4: .1,
+  -5: .06,
+  -6: .02,
 ) !default;
 
+$box-shadow-primary: 0 2px 8px 0;
```

Comment:
Maybe add a comment to clarify that themify adds the colour property.
Or find a way to provide a default colour and override it in themify :smile:

Reply:
Agreed!